app/components/govuk_component/footer_component.html.erb

```diff
@@ -26,7 +26,7 @@
           <% end %>
 
           <% if meta_licence.nil? %>
-            <svg aria-hidden="true" focusable="false" class="<%= brand %>-footer__licence-logo" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 483.2 195.7" height="17" width="41">
+            <svg aria-hidden="true" focusable="false" class="<%= brand %>-footer__licence-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 483.2 195.7" height="17" width="41">
```

Comment:
@frankieroberto it might be worth adding a test for this because I can see someone coming along and reverting this change thinking that it was just a typoe.

Reply:
Done in #462